### PR TITLE
Release INAV-1.6

### PR DESCRIPTION
**Pull request by digitalentity**
Release notes draft - https://github.com/iNavFlight/inav/wiki/1.6-Release-notes

### CODE REVIEW

**Comment by MarcoPTG** [comment]
@digitalentity When the final release will be available? I have a new FC to mount and i want to install the stable version of 1.6.
On the Sp F3 deluxe and Omnibus F3 no issues to report.

**Comment by digitalentity** [comment]
@MarcoPTG we are going to release a 1.6-RC1 early next week (hopefully), the final release is going to be published at end of February.

**Comment by MarcoPTG** [comment]
@digitalentity thanks for info.

File: src/main/navigation/navigation.c [hunk 1]
```diff
@@ -127,7 +127,7 @@ PG_RESET_TEMPLATE(navConfig_t, navConfig,
         .launch_motor_timer = 500,             // ms
         .launch_motor_spinup_time = 100,       // ms, time to gredually increase throttle from idle to launch
         .launch_timeout = 5000,                // ms, timeout for launch procedure
-        .launch_climb_angle = 10              // 10 deg
+        .launch_climb_angle = 18              // 10 deg
```
Comment (martinbudden) [inline src/main/navigation/navigation.c]: Comment needs updating as well.

**Pull request merged**